colcon_cmake/task/cmake/__init__.py

```diff
@@ -206,6 +206,20 @@ def get_project_file(path, target):
     return project_file
 
 
+def get_msbuild_version():
+    """
+    Get the version of msbuild or throw an exception.
+
+    :rtype: str
+    """
+    cp = subprocess.run(
+        ['msbuild', '-version', '-noLogo'],
+        capture_output=True,
+        encoding=sys.getdefaultencoding(),
```

Comment:
The `encoding` argument was only added in Python 3.6 but this package targets platforms still using Python 3.5 (Ubuntu Xenial, Debian Stretch).